storage/mroonga/ha_mroonga.cpp

```diff
@@ -296,6 +297,7 @@ static PSI_mutex_info mrn_mutexes[] =
 #endif
 
 /* global variables */
+bool mrn_initialized = false;
```

Comment:
Is this safe for multi-threading or CPU optimization? Is there a need to add mrn_mutexes similar to other global variables?

Reply:
Yes. It's safe for multi-threading. This is changed only by `st_mysql_plugin::init` and `st_mysql_plugin::deinit` and they aren't called with multiple threads at once.

I'm not sure for CPU optimization. What situation do you think about?